src/Pact/Types/Command.hs

```diff
@@ -301,9 +301,17 @@ instance ToJSON PactResult where
   toJSON (PactResult (Right s)) =
     object [ "status" .= ("success" :: String)
            , "data" .= s ]
-  toJSON (PactResult (Left f)) =
+  toJSON (PactResult (Left pe)) =
     object [ "status" .= ("failure" :: String)
-           , "error" .= f ]
+           , "error" .= perror pe
+           ]
+    where
+      perror (PactError t i _ d) = object
+        [ "type" .= t
+        , "info" .= renderInfo i
```

Comment:
`Info` has two relatively incompatible renderings: this one just does `foo:0:2` etc, whereas `show` displays the code with ellipses. What's interesting is `Info` doesn't have a `Pretty` instance which would be an opportunity to do this right without breaking any other usages ... BTW we probably can't touch this at this point as it would break compat :(

Reply:
:(